.github/workflows/docbuild.yml

```diff
@@ -20,9 +20,7 @@ jobs:
         python -m pip install --upgrade pip
         pip install .[dev]
         pip install "selenium<4.3.0"
```

Comment:
Can't test it right now but maybe we can also remove the installation of selenium

Reply:
Oh, that's a good point. Yeah, selenium shouldn't be needed any more.